### ISSUE TITLE
Sharing button hidden when using an adblocker 

### ISSUE THREAD

**Issue by tomneedham** [owncloud/calendar#235]
### Steps to reproduce
1. Hover over a calendar
### Expected behaviour

Should see share icon
![screen shot 2016-02-25 at 15 01 05](https://cloud.githubusercontent.com/assets/660805/13323576/ff4724b4-dbd1-11e5-85d3-dc9839bedad5.png)
### Actual behaviour

Isnt shown
![screen shot 2016-02-25 at 15 00 53](https://cloud.githubusercontent.com/assets/660805/13323580/045d9406-dbd2-11e5-9f0f-faeb4c044501.png)

Same as contacts https://github.com/owncloud/contactsrework/issues/54

**Comment by georgehrke** [comment]
@tomneedham Do you use some kind of ad-blocker?

**Comment by tomneedham** [comment]
Exactly what I'm checking ;)

**Comment by tomneedham** [comment]
Yup, blocked by https://adblockplus.org. Just trying to see what rule is triggering it.

**Comment by jancborchardt** [comment]
Ah crap, we had something like this in the past for Files as well. We should contact Adblock Plus and provide them with a rule to counter this. @tomneedham once you find out what triggers this can you take care?

**Comment by tomneedham** [comment]
Yeah will do. I thought it would be the class `icon-share` but apparently not.. digging.

**Comment by tomneedham** [comment]
Currently I see this issue with Safari 9.0.3 and Adblock Plus

**Comment by jancborchardt** [comment]
It’s definitely something with the word »share«. Be it in the image or in the CSS.

**Comment by tomneedham** [comment]
https://adblockplus.org/forum/viewtopic.php?f=18&t=44184 tracked here as well whilst I try and debug this.

**Comment by tomneedham** [comment]
It seems 'Fanboy's Social Blocking List' is the culprit at least for me, even though ABP says zero ads are blocked on the page.

**Comment by tomneedham** [comment]
From the forum discussion:
> Reproduced in Firefox and Chrome. Blockable items shows that the offending rule is `##.icon-share` in the `Fanboy's Social Blocking List`.

Reported to the EasyList forum, pending approval from a mod. @jancborchardt Can you confirm which lists you had installed that caused this issue? I've requested that we get icon-share allowed so that all ownCLoud apps can utilise the core icons.

**Comment by georgehrke** [comment]
@tomneedham What's the status on this one?

**Comment by tomneedham** [comment]
It's due to the 'icon-class'. Not sure why this isn't picked up in core, as the rules are basically checking for anything with .icon-class.

**Comment by georgehrke** [comment]
ok, let's move this to 1.1 and tell people to disable their adblocker for ownCloud in the meantime

**Comment by georgehrke** [comment]
@tomneedham Any update?
